doc/processes/continuous_integration/Adding-a-new-Windows-machine-to-Jenkins.md

```diff
@@ -43,16 +98,3 @@ Open an admin cmd window and run `jenkins-agent.exe install` and then `servies.m
 Find the jenkins service in the Service Manager window on the machine, and change it to run as `isis\ibexbuilder` rather than local service account, you'll need to enter `ISISBuilder` password.
 
 Then run start service from the Service Manager window.
-
-### configure jenkins-agent.xml
-
-We need a longer git checkout time than default of ten minutes. To adjust for submodule operations see  https://plugins.jenkins.io/git/#plugin-content-default-timeout  which basically means you need to add `-Dorg.jenkinsci.plugins.gitclient.Git.timeOut=60` to both the jenkins server and the client args. On the java command line put it before the `-jar` option as otherwise it gets passed as an argument to the jar file
-
```

Comment:
Do we need to keep the above information somewhere as it may become relevant again?